lib/text/ui_text_displayer.js

```diff
@@ -35,6 +36,12 @@ shaka.text.UITextDisplayer = class {
   constructor(video, videoContainer, config) {
     goog.asserts.assert(videoContainer, 'videoContainer should be valid.');
 
+    if (!document.fullscreenEnabled) {
```

Comment:
I don't think this is the right check.  This could exist and be false.

Maybe it should be `('fullscreenEnabled' in document)`?